figures/README.md

```diff
@@ -0,0 +1,32 @@
+# Create color palettes and plot theme
+
+### Usage
+
+The `create_color_palette_project.R` script is intended to be run via the command line from the top directory of the repository as follows:
```

Comment:
For extra clarity, maybe you could provide an example of the top directory path?

Reply:
@sharonfreshour The top directory is the `root_dir` for the user. Is this clear or not?

Reply:
This has been now addressed here 5d50679872637b1bafe945e5852306923e164ccf